packages/design-system/src/components/text.tsx

```diff
@@ -57,16 +57,17 @@ export const Text = styled("div", {
       true: {
         whiteSpace: "nowrap",
         textOverflow: "ellipsis",
+        overflow: "hidden",
+
+        // To make sure text is not clipped vertically
+        pt: 10,
```

Comment:
we probably don't need 10px, more like 3-5, but also don't know if that matters in any way

Reply:
oh wait, in px it means its a fixed size so a larger font won't work with this, wouldn't it make sense to use em?